nav2_bringup/rviz/nav2_default_view.rviz

```diff
@@ -58,7 +58,7 @@ Visualization Manager:
         Durability Policy: Volatile
         History Policy: Keep Last
         Reliability Policy: Reliable
-        Value: /robot_description
```

Comment:
Q: I believe if you just save an rviz config, this is how it comes out. While I agree with this change, I worry about its maintainability for the future when new panels are added, configs change, and so on and we save a new config and invariably forget to remove the `/`'s  at some point in the long-term future

Reply:
I gave this a spin and I don't think so, it seems that the rviz box where you fill the topic is just a text box where actually no validation takes place and you can input whatever you want. Saving the rviz config just updates the field, for example I tried to change it such as:

![image](https://github.com/user-attachments/assets/1925f511-61f3-4e58-b3ad-95b1c5f242d8)

And this showed up in my git diff:

![image](https://github.com/user-attachments/assets/3cb38cca-d8fe-4565-9995-bd0d025e0833)